greenframe/sales.js

```diff
@@ -0,0 +1,82 @@
+// eslint-disable-next-line no-unused-expressions
+async page => {
+    const raNotificationAutoHideDuration = 4000;
+    const isRaSettled = () => !document.querySelector('.app-loader');
+    const waitForRaSettled = async () =>
+        await page.waitForFunction(isRaSettled, undefined, {
+            polling: 500,
+            timeout: 5000,
+        });
+    const waitForUndoableNotification = async () =>
+        await page.waitForTimeout(raNotificationAutoHideDuration + 300);
+
+    const addMonths = (date, count) => {
+        if (date && count) {
+            var m,
```

Comment:
Can't you use `const` instead of `var` ? 
Also, maybe rename them as `month` and `futureDate` for example ?